% (c) 2002 Matthew Boedicker <mboedick@mboedick.org> (original author) http://mboedick.org
% (c) 2003-2007 David J. Grant <davidgrant-at-gmail.com> http://www.davidgrant.ca
% (c) 2008 Nathaniel Johnston <nathaniel@nathanieljohnston.com> http://www.nathanieljohnston.com
%
% (c) 2012 Joshua J Hamilton <joshamilton@gmail.com>
%This work is licensed under the Creative Commons Attribution-Noncommercial-Share Alike 2.5 License. To view a copy of this license, visit http://creativecommons.org/licenses/by-nc-sa/2.5/ or send a letter to Creative Commons, 543 Howard Street, 5th Floor, San Francisco, California, 94105, USA.

\documentclass[letterpaper,10pt]{article}
\newlength{\outerbordwidth}
\pagestyle{empty}
\raggedbottom
\raggedright
\usepackage[svgnames]{xcolor}
\usepackage{framed}
\usepackage{tocloft}
\usepackage{verbatim}
\usepackage{fancyhdr}
\usepackage{array}
\usepackage{lastpage}
\usepackage{etaremune}
\usepackage[margin=0.5in]{geometry}
\usepackage[hidelinks]{hyperref}
\renewcommand{\familydefault}{\sfdefault}

% Define custom date format
\usepackage{datetime}
\newdateformat{myDate}{\monthname[\THEMONTH] \THEYEAR}

%-----------------------------------------------------------
%Edit these values as you see fit
\setlength{\outerbordwidth}{3pt}  % Width of border outside of title bars
\definecolor{shadecolor}{gray}{0.75}  % Outer background color of title bars (0 = black, 1 = white)
\definecolor{shadecolorB}{gray}{0.93}  % Inner background color of title bars


%-----------------------------------------------------------
%Margin setup

%\setlength{\evensidemargin}{-0.25in}
\setlength{\headheight}{0in}
\setlength{\headsep}{0in}
%\setlength{\oddsidemargin}{-0.25in}
%\setlength{\paperheight}{11in}
%\setlength{\paperwidth}{8.5in}
\setlength{\tabcolsep}{0in}
%\setlength{\textheight}{9in}
%\setlength{\textwidth}{6.5in}
\setlength{\topmargin}{-0.75in}
\setlength{\topskip}{0in}
\setlength{\voffset}{0.1in}


%-----------------------------------------------------------
%Custom commands
\newcommand{\resitem}[1]{\item #1 \vspace{-2pt}}
\newcommand{\resheading}[1]{\vspace{8pt}
  \parbox{\textwidth}{
  \setlength{\FrameSep}{\outerbordwidth}
\setlength{\fboxsep}{0pt}
\framebox[\textwidth][l]{\setlength{\fboxsep}{4pt}\fcolorbox{shadecolorB}{shadecolorB}{\textbf{\sffamily{\mbox{~}\makebox[7.262in][l]{\large #1} \vphantom{p\^{E}}}}}}
  }
  \vspace{-5pt}
}

% Itemized headings w/ two lines
\newcommand{\ressubheading}[4]{
\begin{tabular*}{7in}{l@{\cftdotfill{\cftsecdotsep}\extracolsep{\fill}}r}
		\textbf{#1} & #2 \\
		\textit{#3} & \textit{#4} \\
\end{tabular*}\vspace{-6pt}}

% Itemized headings w/ one line
\newcommand{\ressubheadingSingular}[2]{
\begin{tabular*}{7in}{l@{\cftdotfill{\cftsecdotsep}\extracolsep{\fill}}r}
		#1 & \textit{#2} \\
%		\textit{#3} & \textit{#4} \\
\end{tabular*}\vspace{-6pt}}


% Item formats
\renewcommand{\labelitemi}{$\bullet$}
\renewcommand{\labelitemii}{$\circ$}
%\renewcommand{\labelitemiii}{$\diamond$}
%\renewcommand{\labelitemiv}{$\ast$}

%-----------------------------------------------------------
% Place information in the footer
\fancypagestyle{plain}{%
\fancyhf{} % clear all header and footer fields
\fancyfoot[L]{Hamilton R\'{e}sum\'{e}}
\fancyfoot[C]{\thepage\ of \pageref*{LastPage}}
\fancyfoot[R]{Last revised: \myDate\today}
\renewcommand{\headrulewidth}{0pt}
\renewcommand{\footrulewidth}{0pt}
\setlength\footskip{0.5in}
}
\pagestyle{plain}

%-----------------------------------------------------------

\begin{document}

{
\centering 
\textbf{\Large Joshua J. Hamilton}  \\
\large{San Francisco Bay Area, CA, USA} \,
\large{joshamilton@gmail.com} \,
\large{913-269-7789} \\
\large{https://www.linkedin.com/in/joshamilton/} \qquad \large{https://github.com/joshamilton}
%\large{\emph{Location}: San Francisco, CA, USA} \\
%\large{\emph{E-mail}: joshamilton@gmail.com} \,
%\large{\emph{Phone}: 913-269-7789} \,
%\large{\emph{Web}: \url{https://www.linkedin.com/in/joshamilton/}} \\
\par
}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\resheading{Professional Summary}
\begin{itemize}
\item Bioinformatician/data scientist with leadership and management experience in cancer immunotherapy and microbiome therapeutics. Expertise in human and microbial bioinformatics, machine learning, software development, and cloud-based systems architecture. Enjoy interdisciplinary collaboration and team-building to discover life-changing therapies for patients. Passionate about educating colleagues in data best practices to facilitate data-driven decision making.
\end{itemize}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\resheading{Experience}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{itemize}

	\item 
		\ressubheadingSingular{\textbf{Senior Bioinformatics Scientist}, Amplify Bio}{March 2024 - December 2024}
		\begin{itemize}
			\item Developed and maintained bioinformatic pipelines to enable neoantigen identification, tumor characterization, and TCR discovery for T-cell receptor therapies
%			\item Performed software validation of custom bioinformatics tools used for neoantigen discovery
			\item Led customization of Benchling LIMS to streamline workflows and enhance data management for programs in TCR discovery and mRNA-based therapeutic development
			\item Developed processes to maintain chain of custody and chain of identity during discovery of personalized TCRs
%			\item Characterized transcriptomic response of TCR-T cells to hypoxic growth, to guide manufacturing for improved anti-tumor potency
		\end{itemize}

	\item 
		\ressubheadingSingular{\textbf{Technical Advisor}, SeqCoast Genomics}{January 2024 - Present}
		\begin{itemize}
			\item Provide strategic guidance on bioinformatics, data pipelines, and cloud-based infrastructure, to enable efficient genomic data processing and delivery to clients
%			\item Developed strategy and coordinated migration of on-prem computing and data storage to AWS
%			\item Evaluated bioinformatics pipelines to support data analysis needs, including assembly, annotation, and variant calling
%			\item Led onboarding of Seqera Platform (a pipeline orchestration tool), including deployment, training, and documentation
%			\item Reviewed, streamlined, and documented data pipeline architectures for new and existing instruments and services
%			\item Developed SOC2 diagrams and data flow documentation to ensure compliance
%			\item Evaluated, designed, and implemented data delivery methods for secure multi-customer access, encompassing testing, deployment, and documentation of S3 bucket policies and AWS Transfer Family
%			\item Established cost tagging and AWS lifecycle policies and to monitor and reduce AWS costs
		\end{itemize}
			
	\item
		\ressubheadingSingular{\textbf{Data Scientist I, Data Scientist II, Sr Data Scientist}, Federation Bio, Inc}{May 2019 - July 2023}
		\begin{itemize}
		
% Leader
			\item Built and led group of three data scientists. Established priorities, managed timelines, and provided mentorship, enabling the team to support multiple research programs in metabolism, immunology, and oncology
			\item Partnered with CMC, translational medicine, clinical science, and regulatory affairs to advance discovery programs into clinical development, by serving as technical expert in microbiome science and data science
			\item Communicated findings to internal and external stakeholders, in the form of research presentations, memos, and regulatory filings

% Contributor: Bioinformatics
			\item Established a functional definition of the healthy human microbiome, and designed the first-ever metabolically-complete synthetic microbiome replacement therapy
			\item Designed novel microbiome therapies with multiple mechanisms of action to correct gut dysbiosis and to treat metabolic and immune-mediated diseases
			\item Provided guidance on experimental design to ensure safety, efficacy, and engraftment of microbiome therapies in human and animal models
			\item Developed and implemented novel algorithms for annotation of bile acid and short-chain fatty acid production, improving FedBio's ability to predict metabolic capabilities of new bacterial strains
			\item Integrated multiple sources of experimental and bioinformatic data to ensure FedBio's bacterial strains were safe for oral administration
			\item Developed machine learning algorithms to detect FedBio's strains in murine and human fecal samples, improving the limit of detection 10-fold and enabling pharmacokinetic analysis of FedBio's microbiome therapies
			\item Validated a GMP-compliant bioinformatic method to confirm identity of master cell banks, drug substance, and drug product, releasing 100s of MCBs over a 9-month period
			\item Led internal and external cross-functional teams in development and qualification of bioanalytical assays, ensuring concentration and composition of FedBio's drug product could be quantified with accuracy and precision
			\item Collaborated on sample collection and statistical analysis plans for Phase 1 trial of FedBio's lead asset, enabling pharmacokinetic analysis and biomarker discovery
			
% Immunology / Oncology
			\item Oversaw analysis of flow cytometry data to characterize immunogenicity of microbial strains
			\item Led development of a machine learning algorithm to predict immune response of microbial strains
			\item Coordinated selection of epitopes to be engineered into antigen-presenting microbial strains

% Contributor: Software / Engineering / Architecture
			\item Collaborated with IT to develop and deploy a scientific computing environment, enabling a team of three data scientists to deliver reproducible and shareable analyses for 100s of projects
			\item Built bioinformatics infrastructure on AWS, enabling terabase-scale analysis of next-generation sequencing datasets
			\item Optimized bioinformatics pipelines to handle a 100\% year-over-year increase with 80\% decrease in turnaround time, and upgraded pipelines to ensure availability of state-of-the-art methods
			\item Oversaw development of 6+ software tools to enable data analysis and visualization by a team of twenty wet-lab scientists
			\item Launched laboratory informatics program using Benchling ELN/LIMS, enabling lineage tracking of 1000s of cell lines
			\item Established templates and training for reporting of study outcomes, thereby accelerating filing of regulatory documentation
		\end{itemize}
		
%	\item
%		\ressubheadingSingular{\textbf{Postdoctoral Research Associate}, University of Wisconsin-Madison}{Sep 2014 - May 2019}
		% Dept of Biochemistry (2018-2019)
		% Dept of Bacteriology (2014-2017)
%		\begin{itemize}
%			\item Mentored graduate students in bioinformatics and software development, enabling reproducible analysis of 10+ manuscripts via publication on Github
%			\item Implemented version-controlled computational workflows using R and the Unix shell. Developed open-source software packages using Python. Performed collaborative software development on GitHub
%			\item Developed microbiome-based processes for production of value-added chemicals from biomass, via multi-omic integration of metagenomic, transcriptomic, and metabolomic datasets with computational metabolic models
%			\item Identified metabolic interactions within environmentally and industrially relevant microbiomes, via multi-omic integration of metagenomic and transcriptomic datasets
%			\item Conducted experiments to identify and manipulate specific molecular interactions within biomedically relevant microbiomes
%			\item Designed microbial communities with high levels of butyrate production, by integrating differential equation modeling of community composition with regression modeling of butyrate production
%			\item Developed new algorithm for improved species-level taxonomic assignment within microbiomes
%			\item Implemented a novel algorithm to predict nutrient requirements from genome-scale metabolic models
%			\item Facilitated meta-analysis of a long-term time series, by defining data model and collaborating with a data engineer to implement data model in an SQL database and ETL the data
%			\item Led development of genomic, transcriptomic, and metabolomic assays, authored laboratory protocols, and identified external service providers and collaborators
%			\item Developed a chemically-defined medium that supports growth of over 20 human microbiome species
%			\item Increased laboratory productivity by 33\% by expanding the anaerobic microbiology lab, including anaerobic chambers and liquid handling robots
%			\item Organized first annual Madison Microbiome Meeting	
%			\item Wrote successful fellowship applications for graduate and post-doctoral studies
%			\item Presented research findings via oral and poster presentation at scientific research conferences
%			\item Published in peer-reviewed journals
%			\item Mentored undergraduate and graduate students to completion and publication of research projects
%			\item Collaborated with an interdisciplinary team to characterize a novel light-driven proton-pumping opsin protein
%		\end{itemize}
%	\item
%		\ressubheadingSingular{\textbf{Graduate Research Assistant}, UW-Madison}{Aug 2009 - Sep 2014}		
%		\begin{itemize}
%			\item Characterized metabolic potential of 6+ biotechnology-relevant organisms by constructing systems biology models (genome-scale metabolic models)
%			\item Explored differences in genotype-phenotype relationships between bacterial strains by developing an algorithm to identify functional differences between metabolic networks
%			\item Conducted genetic engineering experiments to test hypotheses generated by these algorithms
%			\item Identified optimal operating conditions for methane production by a syntrophic microbial community, by incorporating thermodynamic and metabolomic information into metabolic models using chemoinformatic group contribution methods
%			\item Wrote successful fellowship applications for graduate and post-doctoral studies
%			\item Presented research findings via oral and poster presentation at scientific research conferences
%			\item Published in peer-reviewed journals
%			\item Mentored undergraduate and graduate students to completion and publication of research projects
%		\end{itemize}
\end{itemize}



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\resheading{Skills}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{itemize}
	\item Human bioinformatics: bulk and single-cell RNA-Seq (transcriptomics), neoantigen identification and prioritization, T-cell receptor (TCR) and V(D)J sequencing, tumor characterization, variant calling
	\item Microbial bioinformatics: 16S rRNA sequencing, amplicon sequencing, genome assembly, genome annotation, metagenomic assembly and binning, metagenome profiling, RNA-Seq, transcriptomics
	\item Computational biology: differential equation modeling, genome-scale modeling, Lotka–Volterra models, machine learning, systems biology, time-series modeling
	\item Software development: AWS, Bash, Benchling, CI/CD, Docker, Git, Github, Jupyter, Linux, Nextflow, package development, Posit Connect, Posit Workbench, Python, Quarto, R, RStudio, R markdown, Shiny, SQL, testing, VS Code
%	\item Omics: 16S rRNA sequencing, amplicon sequencing, genomics, transcriptomics, RNA-Seq, metabolomics, metagenomics, metatranscriptomics
%	\item Laboratory: anaerobic microbiology, bacterial cell culture, PCR
	\item Soft skills: leadership, cross-functional collaboration, communication, writing, presentations
	\item Drug development: drug discovery, assay development, preclinical research, translational research, early clinical development
	%clinical trials
\end{itemize}

\newpage

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\resheading{Education}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{itemize}
	\item
		\ressubheading{University of Wisconsin-Madison (UW-Madison)}{Madison, WI}{Ph.D., Chemical Engineering}{2014}
	\item
		\ressubheading{Case Western Reserve University (CWRU)}{Cleveland, OH}{B.S., Chemical Engineering}{2009}
\end{itemize}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\resheading{Selected Patents, Posters, and Publications}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{etaremune}[itemsep=-2pt]
	\item Fett, C, Win Y, \textbf{Hamilton J}, Rosoff H, Bronevetsky Y, Garcia C, Wong A, Pan Z, Velazquez VM,  Kunkel EJ, and A Conroy. (2024)  \emph{Adapting T cells for the Tumor Microenvironment (TME) During Manufacturing for Improved Anti-Tumor Potency}. International Society for Cell \& Gene Therapy. 
	 \item Swem LR, Kumar P, Bhalla A, Tripathi SA, Parmar A, \textbf{Hamilton JJ}, Brumbaugh AR, Ricci DP, Layman HRW, Ciglar AM, Berleman J, Walters Z, Jacoby K, Youngblut ND, Grauer A, Drabant Conley E, Romasko H (2023) \emph{Microbial consortia}. \href{https://image-ppubs.uspto.gov/dirsearch-public/print/downloadPdf/20230165913}{US Patent Application No. 18/060,831}.
%	 \item Fischbach MA, Brumbaugh AR, Cheng AG, Dodd D, Aranda-Diaz AJ, Wang M, Yu FB, Sonnenburg JL, Huang KC, Higginbottom S, Jain SS, Meng X, Swem L, Ricci D, \textbf{Hamilton JJ}. (2023) \emph{High-Complexity Synthetic Gut Bacterial Communities}. \href{https://image-ppubs.uspto.gov/dirsearch-public/print/downloadPdf/20230233620}{US Patent Application No. 17/999,516}.
	 \item Swem L, Ricci D, Brumbaugh AR, Cremin J, \textbf{Hamilton JJ}, Tripathi S, Wong L, Romasko H, Bracken R, Drabant Conley E. (2023) \emph{Microbial consortia for the treatment of disease}. \href{https://image-ppubs.uspto.gov/dirsearch-public/print/downloadPdf/20230125976}{US Patent Application No. 17/906,060}.
	\item Ricci D, \textbf{Hamilton JJ}, Tripathi S, Brumbaugh A, Cremin J, Ou N, Layman H, and L Swem. (2022) \emph{Creation of Rationally Designed and Metabolically Active Microbiome Consortia for Treatment of Enteric Hyperoxaluria.} Kidney International Reports. 7(2): S204-S205. \href{https://www.kireports.org/article/S2468-0249(22)00490-9/fulltext}{doi:10.1016/j.ekir.2022.01.490}.
%	\item Feng J, Qian Y, Zhou Z, Ertmer S, Vivas EI, Lan F, \textbf{Hamilton JJ}, Rey FE, Anantharaman K, OS Venturelli. (2022) \emph{Polysaccharide utilization loci in Bacteroides determine population fitness and community-level interactions}. 30(2) P200-215.E12. Cell Host \& Microbe. \href{https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(21)00577-1}{doi:10.1016/j.chom.2021.12.006}.
	\item Clark RL, Connors B, Stevenson DM, Hromada SE, \textbf{Hamilton JJ}, Amador-Noguez D, and OS Venturelli. (2021) \emph{Design of synthetic human gut microbiome assembly and function}. Nature Communications. 12: 3254. \href{https://www.nature.com/articles/s41467-021-22938-y}{doi:10.1038/s41467-021-22938-y}.
	\item Scarborough MJ, \textbf{Hamilton JJ}, Erb EA, Donohue TJ, and DR Noguera. (2020) \emph{Diagnosing and Predicting Mixed-Culture Fermentations with Unicellular and Guild-Based Metabolic Models}. mSystems. 5(5):e00755-20. \href{https://doi.org/10.1128/mSystems.00755-20}{doi:10.1128/mSystems.00755-20}.
	\item Cao X*, \textbf{Hamilton JJ*}, and OS Venturelli. (2019) \emph{Understanding and Engineering Distributed Biochemical Pathways in Microbial Communities}. Biochemistry. 58(2): 94-107. \href{https://doi.org/10.1021/acs.biochem.8b01006}{doi:10.1021/acs.biochem.8b01006}.
%	\item Dwulit-Smith JR, \textbf{Hamilton JJ}, Stevenson DM, He S, Oyserman BO, Moya-Flores F, Amador-Noguez D, McMahon KD, and KT Forest. (2018) \emph{acI Actinobacteria Assemble a Functional Actinorhodopsin with Natively-synthesized Retinal}. Applied and Environmental Microbiology. 84(24): e01678-18. \href{https://doi.org/10.1128/AEM.01678-18}{doi:10.1128/AEM.01678-18}.
	\item Scarborough MJ, Lawson CE, \textbf{Hamilton JJ}, Donohue TJ, and DR Noguera. (2018) \emph{Metatranscriptomic and Thermodynamic Insights into Medium-Chain Fatty Acid Production Using an Anaerobic Microbiome}. mSystems. 3(6): e00221-18. \href{https://doi.org/10.1128/mSystems.00221-18}{doi:10.1128/mSystems.00221-18}.
	\item Rohwer RR, \textbf{Hamilton JJ}, Newton, RJ, and KD McMahon. (2018) \emph{TaxAss: Leveraging a Custom Freshwater Database Achieves Fine-Scale Taxonomic Resolution}. mSphere. 3(5): e00327-18. \href{https://doi.org/10.1128/mSphere.00327-18}{doi:10.1128/mSphere.00327-18}.
%	\item Garcia SL, Buck M, \textbf{Hamilton JJ}, Wurzbacher C, Grossart HP, McMahon KD, and A Eiler. (2018) \emph{Model Communities Hint at Promiscuous Metabolic Linkages between Ubiquitous Free-Living Freshwater Bacteria}. mSphere. 3(3): e00202-18. \href{https://doi.org/10.1128/mSphere.00202-18}{doi:10.1128/mSphere.00202-18}.
	\item \textbf{Hamilton JJ}, Garcia SL, Brown BS$^\dagger$, Oyserman BO, Moya F, Bertilsson S, Malmstrom RR, Forest KT, and KD McMahon. (2017) \emph{Metabolic Network Analysis and Metatranscriptomics Reveals Auxotrophies and Nutrient Sources of the Cosmopolitan Freshwater Microbial Lineage acI}. mSystems. 2(4): e00091-17. \href{https://doi.org/10.1128/mSystems.00091-17}{doi:10.1128/mSystems.00091-17}.
	\item Lawson CE, Wu S, Bhattacharjee AS, \textbf{Hamilton JJ}, McMahon KD, Goel R, and DR Noguera. (2017) \emph{Metabolic network analysis reveals microbial community interactions in anammox granules}. Nature Communications. 8: 15416. \href{https://www.nature.com/articles/ncomms15416}{doi:10.1038/ncomms15416}.
	\item \textbf{Hamilton JJ}, Calixto Contreras M$^\dagger$, and JL Reed. (2015) \emph{Thermodynamics and H$_2$ Transfer in a Methanogenic, Syntrophic Community.} PLoS Computational Biology. 11(7): e1004364. \href{http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004364}{doi:10.1371/journal.pcbi.1004364}.
%	\item Vinay-Lara E, \textbf{Hamilton JJ}, Stahl B, Broadbent JR, Reed JL, and JL Steele. (2014) \emph{Genome-Scale Reconstruction of Metabolic Networks of Lactobacillus casei ATCC 334 and 12A}. PLoS ONE. 9(11): e110785. \href{http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0110785}{doi:10.1371/journal.pone.0110785}.
	\item \textbf{Hamilton JJ} and JL Reed. (2014) \emph{Software platforms to facilitate reconstructing genome-scale metabolic networks}. Environmental Microbiology. 16(1): 49-59. \href{http://onlinelibrary.wiley.com/doi/10.1111/1462-2920.12312/abstract}{doi:10.1111/1462-2920.12312}.
	\item \textbf{Hamilton JJ}, Dwivedi V$^\dagger$, and JL Reed. (2013) \emph{Quantitative Assessment of Thermodynamic Constraints on the Solution Space of Genome-Scale Metabolic Models.} Biophysical Journal. 105(2): 512-522. \href{http://www.cell.com/biophysj/abstract/S0006-3495%2813%2900685-1}{doi:10.1016/j.bpj.2013.06.011}.
	\item \textbf{Hamilton JJ} and JL Reed. (2012) \emph{Identification of Functional Differences in Metabolic Networks Using Comparative Genomics and Constraint-Based Models}. PLoS ONE. 7(4): e34670. \href{http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0034670}{doi:10.1371/journal.pone.0034670}.
\end{etaremune}
* indicates equal contribution \\
$\dagger$ indicates an undergraduate student author

\end{document}